imports/plugins/core/graphql/server/resolvers/account/query/account.js

```diff
@@ -0,0 +1,24 @@
+import { transformIdFromBase64 } from "@reactioncommerce/reaction-graphql-utils";
+import { userAccountQuery } from "/imports/plugins/core/accounts/server/methods/userAccountQuery";
+
+/**
+ * @name account
+ * @method
+ * @summary query the Accounts collection and return user account data
+ * @param {Object} _ - an object containing the result returned from the resolver
+ * @param {Object} args - an object of arguments to pass to the function
+ * @param {String} args.id - id of user to query
+ * @param {Object} context - an object containing the per-request state
+ * @return {Object} user account object
+ */
+export default function account(_, { id }, context) {
+  // Trasform ID from base64
+  // Returns an object. Use `.id` to get ID
+  const idFromBase64 = transformIdFromBase64(id);
+
+  // Pass decoded id (idFromBase64.id) and context into userAccountQuery function
+  const userAccount = userAccountQuery(context, idFromBase64.id);
```

Comment:
On this return object, you'll need to change all of the IDs to the base64 IDs using `transformIdToBase64`. Add `_id` to your graphql query any you'll notice that it doesn't match the one you requested (it's the Mongo one). That's because you didn't transform the result before returning it.